draft/0003-organization-invitation.md

```diff
@@ -0,0 +1,131 @@
+---
+Start date: 2023-12-26
+Author(s): gao
+Revision: 1
+---
+
+# Organization invitation
+
+## 1. Overview
+
+Invitation defines a way to invite a user to join an [organization](./0001-organization.md), and it is a common feature of user collaboration. This specification demonstrates the basics of invitation in Logto, and aims to provide a standard way to implement the feature.
+
+## 2. Motivation
+
+While organizations built the foundation for multi-tenancy, collaboration features are our next step to enable end users to work together. Invitation is a good starting point, as it is a self-contained feature for Logto Cloud, just like organizations.
+
+## 3. Specification
+
+### 3.1. Invitation
+
+We define an invitation as a data structure that contains the following information:
+
+- **Invitation ID**: A unique identifier for the invitation.
+- **Inviter user ID**: The user who created the invitation.
+- **Invitee**: The email address of the invitee.
```

Comment:
How about `Invitee Email`, this is more specific and in the future we may have other identifiers for invitee.

Reply:
thought about this. actually it's rare to use two identifiers for a single invitation since the potential conflict. (e.g. email and phone belong to different accounts) so one single invitee field is ok as long as we can recognize the address.